src/components/DayPickerNavigation.jsx

```diff
@@ -137,9 +146,10 @@ function DayPickerNavigation({
               ] : []),
             ] : []),
           )}
+          aria-disabled={disablePrev}
           aria-label={phrases.jumpToPrevMonth}
-          onClick={onPrevMonthClick}
-          onKeyUp={(e) => {
+          onClick={disablePrev ? undefined : onPrevMonthClick}
+          onKeyUp={disablePrev ? undefined : (e) => {
             const { key } = e;
             if (key === 'Enter' || key === ' ') onPrevMonthClick(e);
           }}
```

Comment:
should the `onMouseUp` similarly be disabled when `disablePrev` is set?

Reply:
The `onClick` covers onMouse down/up on the same element. Is there a scenario where it would only get the `onMouseUp` event ?